### ISSUE TITLE
Hotspot- nº usuarios y ancho de banda

### ISSUE THREAD

**Issue by albertpardo** [ethereum-internet-access/Captive-Portal#4]
Por lo que he leido sobre hostspots, se comenta  el número máximo de usuarios a que da servicio y el ancho de banda asignado a cada uno.

Creo que en nuestra PoC con rasberryPi deberiamos indicar esta cifra.

**Comment by lluisgv2** [comment]
Estoy de acuerdo en que se tiene que mencionar esta cuestión. Pero, ¿vamos a ponernos a calcular ésto? En mi opinión se nos va del ámbito. Propongo que nos limitemos a indicar que en esta PoC no vamos a limitar ni administrar estos parámetros, y que delegamos esta cuestión al router al que está conectada la Raspberry Pi.

**Comment by albertpardo** [comment]
Creo que será mejor que hagamos test de velocidad de la conexión. Conectar uno , dos  o más portátiles por la wifi a la rasberry y que cada portatil ejecute su test de velocidad de la conexión desde el navegador. 

Como ejemplo para mirar el test de velocidad : [Test Velocidad Movistar](http://www.movistar.es/particulares/test-de-velocidad/medir-velocidad)

Quizas conozcais otras webs para medir la velodidad que sean más fiables.
